.github/workflows/wheels.yml

```diff
@@ -105,10 +110,17 @@ jobs:
             py: cp310
             arch: universal2
 
+          # Windows-latest cp310, we can only include win_amd64 builds as numpy
+          # does not support win
```

Comment:
This looks like this sentence was truncated and there's something missing here.

Reply:
Will fix